demos/README.md

```diff
@@ -280,3 +280,16 @@ The demo notebooks can be run without any installation of Python by using Binder
 <!-- DEMO TABLE MARKER -->
 
 See [the root README](../README.md) or each algorithm's documentation for the relevant citation(s).
+
+## Download the demos
```

Comment:
FWIW, https://github.com/stellargraph/stellargraph/pull/1433 removes content like this from this README. Do you think this is still valuable to include even with a switch to RtD for most things? (#1433 is designed to allow custom content like this.)

Reply:
I think it's still useful to know how to download all of the demos; downloading each one separately from readthedocs is less convenient in many cases.

Reply:
Ah I see, if we're linking to RTD, I don't think "downloading demos" is a particularly important custom content to add. Should I remove it from this PR?

Reply:
> I think it's still useful to know how to download all of the demos

It'll be included in the readthedocs `demos/index.rst` still, but the question is around whether we also want to include it in `demos/README` right?

Reply:
Aaah, I see.  After looking at the output, I think just the base README is enough.

Reply:
> is that being fixed in a different PR @huonw?

Yup: https://github.com/stellargraph/stellargraph/pull/1448 (https://stellargraph--1448.org.readthedocs.build/en/1448/README.html#getting-started)